styles/style.css

```diff
@@ -209,6 +276,7 @@ p {
 
 .facebook-link {
   color: var(--orange-dark);
+  margin-left: 11.5em;
```

Comment:
I wouldn't use a specific value to right-justify an element. Screen sizes may vary and it may not be 11.5 em to right-justify the FB link. Since this is a text, you can use text-align on the parent div (which needs to be created)

Reply:
Thanks Amellie, I created the div, gave it a class of facebook and aligned right.